apps/quanta/app/utils.ts

```diff
@@ -0,0 +1,17 @@
+import config from '@plone/registry';
+
+/**
+ * Flatten to app server URL - Given a URL if it starts with the API server URL
+ * this method flattens it (removes) the server part
+ * TODO: Update it when implementing non-root based app location (on a
+ * directory other than /, eg. /myapp)
+ * @method flattenToAppURL
+ */
+export function flattenToAppURL(url: string | undefined): string | undefined {
```

Comment:
I would argue that with our current policy of handling all the backend requests through our server, we don't need this anymore. Or, better, we should make it so that we don't need this.